src/meta/src/hummock/manager/compaction/compaction_group_schedule.rs

```diff
@@ -76,6 +77,41 @@ impl HummockManager {
         assert!(member_table_ids_1.is_sorted());
         assert!(member_table_ids_2.is_sorted());
 
+        let created_tables = if let Some(created_tables) = created_tables {
+            // if the created_tables is provided, use it directly, most for test
```

Comment:
I think it is **only** for test? How hard it is to init catalog in the tests? I think the tests already have a in-memory meta store initialized. If it is too hard, we should at least add an assertion to make sure this code path can only be reached in tests.

Reply:
Disscuss with Shanicky, Currently hooking data directly to sqlbackend is not standardised. 
I chose to provide two separate interfaces for ut
1. merge_compaction_group(id1, id2)
2. merge_compaction_group_for_test(id1, id2, created_table)

And add assert to ensure that only ut will use the created_table passed in. I'll remove this implementation in the future when there's a better way to hook data.